webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -186,15 +186,26 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
   const onSortChange = useCallback(
     (sorts: Sort[]) => {
       setSorts(sorts);
+      updateSettings({
+        // save sorts to settings:
+        sorts: JSON.stringify(sorts),
+      });
```

Comment:
consider reading/writing the `sortString` instead -- it only keeps the valid sort entries.